config/filter.d/slapd.conf

```diff
@@ -0,0 +1,25 @@
+# slapd (Stand-alone LDAP Daemon) openldap daemon filter
+#
+# Detecting invalid credentials: error code 49
+# http://www.openldap.org/doc/admin24/appendix-ldap-result-codes.html#invalidCredentials (49)
+
+[INCLUDES]
+
+# Read common prefixes. If any customizations available -- read them from
+# common.local
+before = common.conf
+
+[Definition]
+
+_daemon = slapd
+
+failregex = ^(?P<__prefix>%(__prefix_line)s).* conn=(?P<pid>\d+) fd=\d+ ACCEPT from IP=<HOST>\:\d+ .+$<SKIPLINES>(?P=__prefix).+ conn=(?P=pid) .* RESULT .* err=49 .*$
```

Comment:
- too many greedy catch-alls (bad performance, high load, apart from that dangerously, because fault-prone);
- catch-all `.*` directly after prefix line is not required;
- anchored regex `.*$` - such expressions make no sence at all;
- `pid` means connection id here;

So I'll suggest:
```diff
- failregex = ^(?P<__prefix>%(__prefix_line)s).* conn=(?P<pid>\d+) fd=\d+ ACCEPT from IP=<HOST>\:\d+ .+$<SKIPLINES>(?P=__prefix).+ conn=(?P=pid) .* RESULT .* err=49 .*$
+ failregex = failregex = ^(?P<__prefix>%(__prefix_line)s)conn=(?P<_conn_>\d+) fd=\d+ ACCEPT from IP=<HOST>[^\n]+<SKIPLINES>(?P=__prefix)conn=(?P=_conn_) op=\d+ RESULT(?:\s(?!err)\S+=\S*)* err=49\s
```
Expression `(?:\s(?!err)\S+=\S*)*` means - any count of "tags" in form of `tag=?anything?`, but only if tag not equal "err", so will match ` a=1 b=2 c= ...` or nothing before `err=...`.